src/black/trans.py

```diff
@@ -1062,6 +1062,7 @@ def _prefer_paren_wrap_match(LL: List[Leaf]) -> Optional[int]:
             syms.listmaker,
             syms.dictsetmaker,
             syms.testlist_gexp,
+            syms.arglist,
         ]
         # If the string is an immediate child of a list/set/tuple literal...
```

Comment:
Should update this comment.

Also, since we're now adding parens in basically all contexts, could we get rid of `matching_nodes` entirely?

Reply:
Done, and yes we no longer need `matching_nodes`.